### PR TITLE
100DaysOfGatsby - Challenge 2 - Build and Deploy

### PR DESCRIPTION
**Pull request by hashimwarren**
This challenge asks participants to build and host their Gatsby site using free tools.

### CODE REVIEW

File: docs/blog/100days/free-hosting/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+_To help you learn about Gatsby and our ecosystem, we invite you to take the #100DaysOfGatsby challenge! If you are new to Gatsby, you can follow along with our beginner-friendly weekly prompts. Or if you are familiar with Gatsby and want to go deeper, we invite you to [build your first Gatsby Theme](/docs/themes/building-themes/)._
+
+_Learn more about [#100DaysOfGatsby here](/blog/100days)!_
+
+## Challenge 2: Build and Deploy Your Gatsby Site (Using Free Tools)
+
+This week we invite you to deploy your Gatsby project to the web!
+
+If you’re experienced with launching an all-in-one CMS-powered or database-backed website, then the process of putting a Gatsby site live on the web may feel strange to you. You will be using continuous deployment tools to prepare and publish your site. It’s okay if your site is unfinished, as getting a website deployed will allow you to test on mobile devices and share with your colleagues to get feedback in design and development.
```
Comment (laurieontech) [inline docs/blog/100days/free-hosting/index.md]:
```suggestion
If you’re experienced with launching an all-in-one CMS-powered or database-backed website, then the process of putting a Gatsby site live on the web may feel strange to you. You will be using continuous deployment tools to prepare and publish your site. It’s okay if your site is unfinished, as getting a website deployed will allow you to test on mobile devices and share with your colleagues to get feedback on design and development.
```

File: docs/blog/100days/free-hosting/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+To complete this challenge, we would like you to:
+
+1. host your project in a cloud git repository (publicly or privately),
+2. then use a “build service” to create a production version of your site
+3. and automatically publish your site’s assets directly to a CDN.
```
Comment (laurieontech) [inline docs/blog/100days/free-hosting/index.md]:
```suggestion
3. and automatically publish your site’s assets directly to a Content Delivery Network (CDN).
```

File: docs/blog/100days/free-hosting/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Amazingly, for small projects you can use all of these services for free. There are many Gatsby sites in the wild that pay \$0 for hosting!
+
+Bonus: Try deploying your site to the free tier of multiple services to see first hand which one you’d like to adopt. You may notice differences in features, developer experience, and build speeds and limits.
+
+### Challenge Resources
+
+For your git repo, you can use GitHub, Gitlab, Bitbucket, or many similar services.
+
+And for your build tool and CDN we see many people have success with [AWS Amplify](/docs/deploying-to-aws-amplify/) and [Netlify](/docs/deploying-to-netlify/). Our documentation has step-by-step tutorials for services, [and others].
```
Comment (laurieontech) [inline docs/blog/100days/free-hosting/index.md]:
```suggestion
And for your build tool and CDN we see many people have success with [AWS Amplify](/docs/deploying-to-aws-amplify/) and [Netlify](/docs/deploying-to-netlify/). Our documentation has step-by-step tutorials for those services, [and others].
```
Should there be a URL for the and others piece?

Comment (hashimwarren) [reply docs/blog/100days/free-hosting/index.md]: added link to doc on all deploy services 👍🏾

**Pull request merged**